plugins/completion/coq-thirdparty.nix

```diff
@@ -0,0 +1,65 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.coq-thirdparty;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.coq-thirdparty = {
```

Comment:
We should add the extraOptions option right ?

Reply:
The `require('coq_3p')` function only takes a list of sources as input. There is no such thing as a table of options.